schemas/bridging/broadcastRequest.schema.json

```
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/bridging/broadcastRequest.schema.json",
+  "title": "BroadcastRequest",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "bridgeRequest.schema.json"
+    },
+    {
+      "type":"object",
+      "properties": {
+        "type": {
+          "const": "broadcastRequest"
+        },
+        "payload": {
+          "type": "object",
+          "properties": {
+            "channel": {
+              "$ref": "../api/channel.schema.json"
+            },
+            "context": {
+              "$ref": "../context/context.schema.json"
+            }
+          },
+          "additionalProperties": false,
+          "required": ["channel", "context"]
+        },
+        "meta": true
```

I think this needs refining to *only* come from an app
```suggestion
        "meta": {
          "allOf": [
            {
              "$ref": "bridgeRequest.schema.json#/$defs/RequestMeta"
            },
            {
              "properties": {
                "destination": {
                  "$ref": "../api/appIdentifier.schema.json"
                }
              }
            }
          ]
        }
```